scripts/docker-compose.yml

```diff
@@ -208,7 +208,7 @@ services:
       - host.docker.internal:host-gateway
 
   endorser-agent:
-    image: bcgovimages/aries-cloudagent:py36-1.16-1_0.8.1
+    image: ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.8.2
```

Comment:
Might want to keep the image versions consistent between agent and endorser

Reply:
Right, I had done that locally and never committed. Fixed for local docker. PRs and envs are still pointing to tools endorser, which is an older version as well (not even 0.8.2), but those should be pointed to other endorsers anyways soon.